### PR TITLE
Update and tidy poms

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -17,49 +17,25 @@
...
-    <modelVersion>4.0.0</modelVersion>
     <groupId>jakarta.servlet.jsp.jstl</groupId>
     <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
-    <packaging>jar</packaging>
     <version>2.0.0-SNAPSHOT</version>
-    <name>JavaServer Pages(TM) Standard Tag Library API</name>
+    <packaging>jar</packaging>
 
-    <description>JavaServer Pages(TM) Standard Tag Library API</description>
+    <name>Jakarta Server Pages Standard Tag Library API</name>
```
Comment (pnicolucci) [inline api/pom.xml:36]: Can we just make this "Jakarta Standard Tag Library API"?

File: impl/pom.xml [hunk 2]
```diff
@@ -17,49 +17,25 @@
...
-    <modelVersion>4.0.0</modelVersion>
     <groupId>org.glassfish.web</groupId>
     <artifactId>jakarta.servlet.jsp.jstl</artifactId>
     <version>2.0.0-SNAPSHOT</version>
     <packaging>jar</packaging>
-    <name>JavaServer Pages (TM) TagLib Implementation</name>
 
-    <description>JavaServer Pages(TM) Standard Tag Library API</description>
+    <name>Jakarta Server Pages TagLib Implementation</name>
+    <description>Jakarta Server Pages Standard Tag Library API</description>
```
Comment (pnicolucci) [inline impl/pom.xml:37]: Also change to "Jakarta Standard Tag Library API" ? Since that matches the specification name.

File: impl/pom.xml [hunk 3]
```diff
@@ -277,147 +378,67 @@ Copyright &#169; 2019 Eclipse Foundation. All rights reserved.
...
-                                    <includes>
-                                        <include>NOTICE.md</include>
-                                        <include>LICENSE.md</include>
-                                    </includes>
-                                    <targetPath>META-INF</targetPath>
-                                </resource>
-                            </resources>
+                            <groups>
+                                <group>
+                                    <title>Jakarta Server Pages Tag Libraries Documentation</title>
```
Comment (pnicolucci) [inline impl/pom.xml:390]: Same comment, match spec name.

File: spec/pom.xml [hunk 4]
```diff
@@ -16,65 +16,67 @@
...
 
+    <properties>
+        <site.output.dir>${project.build.directory}/staging</site.output.dir>
+        <maven.site.skip>true</maven.site.skip>
+        <asciidoctor.maven.plugin.version>1.5.7.1</asciidoctor.maven.plugin.version>
+        <asciidoctorj.version>2.3.0</asciidoctorj.version>
+        <asciidoctorj.pdf.version>1.5.3</asciidoctorj.pdf.version>
+        <jruby.version>9.2.11.1</jruby.version>
+        <!-- status: DRAFT, BETA, etc., or blank for final -->
+        <status>Final Release</status>
```
Comment (pnicolucci) [inline spec/pom.xml:56]: I have this as DRAFT in my PR so we might get a merge conflict here that we'll need to resolve in on of these PRs.

File: api/pom.xml (hunk 1 above)
Comment (arjantijms) [reply api/pom.xml]: Yes, done!

File: impl/pom.xml (hunk 3 above)
Comment (arjantijms) [reply impl/pom.xml:390]: Done

File: spec/pom.xml (hunk 4 above)
Comment (arjantijms) [reply spec/pom.xml:56]: Yes, I'll resolve the merge conflict. Should be easy.

**Comment by arjantijms** [comment]
> Will this PR resolve that?

Yes, it builds with JDK 11 to 13 at least. It's essentially updating all the versions, nothing really major.

**Comment by pnicolucci** [comment]
@arjantijms  Ln 390 of impl/pom still has:

       <title>Jakarta Server Pages Tag Libraries Documentation</title>

If you can update this then we'll  be good to go.

**Comment by arjantijms** [comment]
Thx!

**Pull request merged**